src/libs/utils.ts

```diff
@@ -10,34 +10,46 @@ import { v4 as uuid } from 'uuid'
 import { getCloudProviderFromWorkspace } from './workspace-utils'
 
 
-// TODO: add good typing (remove any's) - ticket: https://broadworkbench.atlassian.net/browse/UIE-67
-export const subscribable = () => {
-  let subscribers = []
+export interface Subscribable<T extends any[]> {
+  subscribe: (fn: (...args: T) => void) => { unsubscribe: () => void }
+  next: (...args: T) => void
+}
+
```

Comment:
it would be great to add a jsdoc block to at least describe what subscribable does and intended use